problems/looping-through-arrays/problem_nb-no.md

```diff
@@ -0,0 +1,49 @@
+---
+
+# ITERERE GJENNOM ARRAYER
```

Comment:
Liste hørtes mye mer forståelig ut ihvertfall, dog virker det som de lærde sier et array, en tabell eller matrise. Hvilket alternativ stemmer du for @Torsph?